src/index.js

```diff
@@ -0,0 +1,89 @@
+import { editTodo, removeTodo } from './edit-remove.js';
+import { getFromLocal, updateLocalStorage } from './local-storage';// eslint-disable-line
+import './style.css';
+
+// Queries to HTML
+const inputText = document.querySelector('input');
+const todoMainContainer = document.querySelector('.todos-container');
+const ClearBtn = document.querySelector('button');
+
+// class object
+class MyObjects {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export const myArray = [];
+
+// Add Method
+const addTodo = (todoValue) => {
+  const todoContainer = document.createElement('div');
+  todoContainer.className = 'todoContainer';
+  todoContainer.innerHTML += `
+    <input type='checkbox' class='checkbox'>
+    <span>${todoValue}</span>
+    <i class='fas fa-ellipsis-v'></i>
+    <i class='fas fa-trash-alt'></i>
+  `;
+  todoMainContainer.appendChild(todoContainer);
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.toggle('checkedContainer');
+      i.nextElementSibling.classList.toggle('checkToDo');
+      i.parentElement.lastElementChild.classList.toggle('trash-active');
+      i.parentElement.lastElementChild.previousElementSibling.classList.toggle('edited-disable');
+      updateLocalStorage();
+    });
+  });
```

Comment:
- [ ] As required the property `index` is set to the value of the new array length (i.e. if a 1rst task is added to the list, the index of that task should equal to 1, or if a 3rd task is added to the list, the index of that task should equal to 3 ) but is not the case for you, the index starts by 0,1,2,... Please kindly fix that 🙃 😉

<img width="434" alt="Screen Shot 2022-11-02 at 22 15 22" src="https://user-images.githubusercontent.com/74501523/199598463-d824c415-cde7-4213-b53f-f29c4193f4d1.png">